src/gif/fdcExperience.ts

```diff
@@ -128,3 +129,17 @@ export async function generateOperation(
   );
   return res.body.output.messages[0].content;
 }
+
+/**
+ * extractCodeBlock extracts the code block from the generated response.
+ * @param text the generated response from the service.
+ * @return the code block from the generated response.
+ */
+export function extractCodeBlock(text: string): string {
```

Comment:
We should probably get a test for this if possible

Reply:
Sure, just vaib coded a test case for it.